keyboards/maartenwut/wonderland/config.h

```diff
@@ -0,0 +1,55 @@
+/*
+Copyright 2012 Jun Wako <wakojun@gmail.com>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xFEED
+#define PRODUCT_ID      0xA71C
+#define DEVICE_VER      0x0003
+#define MANUFACTURER    Maartenwut
+#define PRODUCT         Wonderland
+#define DESCRIPTION     "QMK Firmware for the Wonderland"
```

Comment:
Afaict, this is what will show when plugging in the keyboard. I would rather name it something like "Wonderland Keyboard" or put keyboard in the name.

Reply:
`DESCRIPTION` is actually only compiled in if you have Command enabled, and it's not used as part of the device name.

This is the only reference to it in the entire repo:
https://github.com/qmk/qmk_firmware/blob/c0e6734d3fc7f777d60ad7bb31002521e9a1ee26/tmk_core/common/command.c#L176-L183
It *was* previously used as part of the device name for Adafruit Bluetooth boards, but for the exact reason you state, it was removed a while back (#5028).

Reply:
This is what the vendor asked for, so I'm not going to change it without their approval. As fauxpark pointed out, it's only used in the COMMAND version feature.